pkg/console/subresource/consoleserver/types.go

```diff
@@ -79,3 +79,12 @@ type Customization struct {
 type Providers struct {
 	StatuspageID string `yaml:"statuspageID,omitempty"`
 }
+
```

Comment:
@pedjak fyi, we should keep this file in sync if you make future changes to the console config.